internal/cli/quickstarts.go

```diff
@@ -31,15 +40,21 @@ var (
 		}
 		return
 	}()
-)
 
-// QuickStart app types and defaults
-const (
-	qsNative    = "native"
-	qsSpa       = "spa"
-	qsWebApp    = "webapp"
-	qsBackend   = "backend"
-	_defaultURL = "http://localhost:3000"
+	clientID = Flag{
+		Name:       "Client ID",
+		LongForm:   "client-id",
+		ShortForm:  "c",
+		Help:       "Client Id of an Auth0 application.",
+		IsRequired: true,
+	}
+
+	stack = Flag{
+		Name:      "Stack",
+		LongForm:  "stack",
+		ShortForm: "s",
+		Help:      "Tech/Language of the quickstart sample to download.",
```

Comment:
Shouldn't it be required as well? (prompt not withstanding)

Reply:
yes, good catch. I was honoring the previous setup but it doesn't make sense without the stack.